services/docker-compose-build.yml

```diff
@@ -27,6 +28,8 @@ services:
         org.opencontainers.image.source: "${VCS_URL}"
         org.opencontainers.image.revision: "${VCS_REF}"
 
+  # STACK ---------------------------
```

Comment:
I am not sure this kind of comments help in anyway. especially with your comment above and the fact that it is below the ```# STACK...``` comment

Reply:
will clarify. All the services below this line will be part of the stack while the ones above are just there to be built as images (e.g. OOIL)